app/controllers/sessions_controller.rb

```diff
@@ -0,0 +1,7 @@
+class SessionsController < Devise::SessionsController
+  def new
+    redirect_to user_companies_find_path and return if Current.company.nil?
+
```

Comment:
remove empty line

Reply:
this line is required by rubocop. "Add empty line after guard clause"